.github/workflows/build.yaml

```diff
@@ -23,6 +23,8 @@ jobs:
       #   run: make lint-go
       - name: go build
         run: go build -v ./...
+      - name: go fmt
+        run: go fmt ./...
```

Comment:
This will format the source but will not fail if the are not formatted. We do want them to fail. 

Instead, we should probably use `make lint` or `make lint-go` 👼🏼

Reply:
ah make sense
Added `fmt` target

Reply:
It doesn't change. It will format the files in the CI, but it won't fail if the files changed. You can test that out with removing the diff on `catalog.go`. The idea here is to fail is the format is not correct.

Reply:
Yes 
it was my very blunder mistake :grin: 

I have modified & tested locally 
This is how it looks
![Screenshot from 2024-03-20 11-54-38](https://github.com/openshift-pipelines/catalog-cd/assets/9441662/299dd114-8612-4910-a18a-fa90bc55bf8e)